Automate/Speckle.Automate.Sdk/Schema/FunctionRunDataParser.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Runtime.Serialization;
+using Speckle.Newtonsoft.Json;
+
+namespace Speckle.Automate.Sdk.Schema;
+
+public static class FunctionRunDataParser
+{
+  /// <summary>
+  /// Function run data parser from json file path./>
+  /// </summary>
+  /// <param name="inputLocation"> Path to retrieve function run data.</param>
+  /// <typeparam name="T"> Type for function inputs.</typeparam>
+  /// <returns>The data to be able run function.</returns>
+  /// <exception cref="SerializationException"> Throws if deserialized object is null.</exception>
+  public static FunctionRunData<T> FromPath<T>(string inputLocation)
+  {
+    string inputJsonString = ReadInputData(inputLocation);
+    FunctionRunData<T>? functionRunData = JsonConvert.DeserializeObject<FunctionRunData<T>>(inputJsonString);
```

Comment:
Note that `FromPath` can also throw a `FileNotFoundException` coming from the call `ReadInputData(...)` on line 17.
The xml docs should reflect this

Reply:
good catch

Reply:
Added. Resolving conservation